js/switcher.js

```diff
@@ -172,6 +178,12 @@ class Switcher extends Component {
     this.initializeState();
   }
 
+  componentDidUpdate(previousProps, previousState, previousContext) {
+    if (!previousState.show && this.state.show) {
+      this.inputNode.focus();
+    }
+  }
+
```

Comment:
Previous solution with `autofocus: true` didn't work in chrome and firefox, this is what I came up with, maybe you know a better way, or it's ok?